requirements-dev.txt

```diff
@@ -6,7 +6,7 @@ mypy>=0.800,<0.900
 openapi-core==0.14.2
 pytest==6.1.0
 pytest-cov==2.11.1
-pytest-django==3.10.0
+pytest-django==4.4.0
```

Comment:
Seemed like the easiest way to get the `capture_on_commit_callbacks` fixture, if that doesn't work (let's see in CI) I guess I'll just move @wedamija's polyfill to make it available in function-based tests.